src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,88 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
 public class SalaryInfo {
+    public static final String DATE_FORMAT = "dd.MM.yyyy";
+
+    private LocalDate dateFrom;
+    private LocalDate dateTo;
+    private String[] names;
+    private int[] salarySum;
```

Comment:
And also there is a rule that service classes must be stateless.
Let's move those fields to the method level